test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -149,7 +149,7 @@ intervals:
   default/wait-machine-status: ["20m", "10s"]
```

Comment:
I looked into the gathered artifacts and found this:

```
- lastTransitionTime: "2024-11-21T20:06:09Z"
    message: |-
      addon_update: updating eks addon coredns: ResourceInUseException: Addon coredns cannot be updated as it is currently in UPDATING state
      {
        RespMetadata: {
          StatusCode: 409,
          RequestID: "776223f5-f6d1-4a1b-83bf-6455dbfc09f6"
        },
        AddonName: "coredns",
        ClusterName: "eks-nodes-kapm37_eks-nodes-8g7yso-control-plane",
        Message_: "Addon coredns cannot be updated as it is currently in UPDATING state"
      }
```

from https://storage.googleapis.com/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_cluster-api-provider-aws/5215/pull-cluster-api-provider-aws-e2e-eks/1859655615442325504/artifacts/clusters/bootstrap/resources/eks-nodes-kapm37/AWSManagedControlPlane/eks-nodes-8g7yso-control-plane.yaml

So I see it's updating CoreDNS.

In this file, it's set to `v1.11.1-eksbuild.8`.

Amazon has versions for a given Kube version here: https://docs.aws.amazon.com/eks/latest/userguide/managing-coredns.html#coredns-add-on-update

For Kube 1.30, it should be `v1.11.3-eksbuild.2`.

I'm going to add a commit to this to see if incrementing the coredns version will help.

Reply:
Looks like this didn't help like I was hoping. For historic reference, https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_cluster-api-provider-aws/5215/pull-cluster-api-provider-aws-e2e-eks/1860077544150142976 is the test run that happened w/ `v1.11.3-eksbuild.2` CoreDNS